### DISCUSSION TITLE
Is there a copy/paste-able list awailable for  'commands' in modules.php?

### DISCUSSION THREAD

**Discussion by berghxyz** [nWidart/laravel-modules#1977] [Q&A] [answered]
Do I have to look for each command and put them in manually or is there a list?

When upgrading from 10 to 11 it seems like I have to find what subfolder each of the commands are put in.
example: 
Commands\CommandMakeCommand::class, -> Commands\Make\CommandMakeCommand::class,

Is there something I have missed or?

**Comment by dcblogdev** [comment, marked as answer]
You can see the internal commands at https://github.com/nWidart/laravel-modules/blob/v11.1.5/src/Providers/ConsoleServiceProvider.php

to see all module commands you can run: `php artisan list | grep module`

**Reply by berghxyz** [reply]
Thank you :)
